src/openzaak/utils/apps.py

```diff
@@ -33,3 +37,12 @@ def ready(self):
 
         # register one-to-one field (for multi-table inheritance of Zaak)
         HANDLERS[models.OneToOneField] = FKHandler
+
+
+def default_user_agent(name=settings.USER_AGENT):
```

Comment:
The `name` argument won't really be used, so you could just return `settings.USER_AGENT` directly (or do `utils.default_user_agent = lambda settings.USER_AGENT`, but it's not a big deal, so I'll approve